src/python/gem5/components/processors/abstract_core.py

```diff
@@ -120,3 +120,35 @@ def get_mmu(self) -> BaseMMU:
         This is used in the board to setup system-specific MMU settings.
         """
         raise NotImplementedError
+
+    @abstractmethod
+    def set_simpoint(
+        self, inst_starts: List[int] = [], init: bool = True
```

Comment:
Why do these parameters have default values? Are neither of them required? `inst_starts` seems required at a minimum.